spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestSnapshotTableProcedure.java

```diff
@@ -103,13 +102,13 @@ public void testSnapshotWithProperties() throws IOException {
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
-  @Test
+  @TestTemplate
   public void testSnapshotWithAlternateLocation() throws IOException {
-    Assume.assumeTrue(
-        "No Snapshoting with Alternate locations with Hadoop Catalogs",
-        !catalogName.contains("hadoop"));
-    String location = temp.newFolder().toString();
-    String snapshotLocation = temp.newFolder().toString();
+    assumeThat(catalogName)
+        .as("No Snapshoting with Alternate locations with Hadoop Catalogs")
+        .doesNotContain("hadoop");
+    String location = temp.toFile().toString();
+    String snapshotLocation = temp.toFile().toString();
```

Comment:
both now point to the same location, whereas in the previous version of the code they were pointing to different folders, so I think we should fix that